.gitignore

```diff
@@ -110,3 +110,14 @@ dmypy.json
 
 # OSX files
 .DS_Store
+
```

Comment:
For this, I think we should add to the blocklist the examples/wip_notebooks, but I don't think it's right to use a gitignore for the others.  They will occasionally need to be updated.  If you're using them, I would recommend just avoiding using `git add` on them.

Reply:
Removed them from the git ignore, the wip folder should be right above it in the list